ansible/roles/nginx/defaults/main.yml

```diff
@@ -8,13 +8,19 @@ nginx_docker_container: nginx
 nginx_workdir: "/docker/nginx"
 nginx_virtualhosts_workdir: "{{ nginx_workdir }}/conf"
 
-virtualhost:
-  fqdn: example.com
+instances:
+  - virtualhost:
+    fqdn: example.com
+    opensearch_dashboards_endpoint: "https://{{ hostvars[(groups['opensearch_dashboards'][0])].ansible_default_ipv4.address }}:5601"
+    host: "{{ hostvars[(groups['opensearch_dashboards'][0]}}"
```

Comment:
We also need to document how to configure this on the ansible documentation files.